drizzle-orm/src/gel-core/dialect.ts

```diff
@@ -149,7 +149,11 @@ export class GelDialect {
 		return sql.join(columnNames.flatMap((colName, i) => {
 			const col = tableColumns[colName]!;
 
-			const value = set[colName] ?? sql.param(col.onUpdateFn!(), col);
+			let value = set[colName];
+			if (!value) {
```

Comment:
this check should be `value === undefined` (across all dialects)

Reply:
oops, just fixed it